nodejs_conductor/index.js

```diff
@@ -137,7 +126,7 @@ Conductor.withInstances = function (instances, opts=defaultOpts) {
 Conductor.run = function (instances, opts, fn) {
     if (typeof opts === 'function') {
         fn = opts
-        opts = {}
+        opts = undefined
```

Comment:
this one because if you pass in an empty object, it won't pick the default args, whereas it will if you pass `undefined`